README.md

```diff
@@ -100,7 +115,7 @@ SQL database. If you need an example, you can use: people and wishes
 
 Your answer:
 ```
-Replace this with your answer
+A schema is a set of ruby object definitions that outline the type of data that will be used for each attribute of a ruby object.  objects are tracked in the database by assigning them ids, defined as SERIAL PRIMARY KEY data that is auto-assigned by the SQL database.  Relationships between data is mapped in two ways. One, by assigning Foreign Key References (the id of a related object, to which the instance object belongs). Two, by models in which classes are assigned whether their instances "has_many" of another class-type or "belongs_to" an instance of another class.
```

Comment:
Great description! `has_many` and `belongs_to` still require foreign keys, though. For instance, if a Person has many Comments, the Comments table needs a `person_id` column.